src/components/BoardView.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div class="board-view">
+    <h3>Board of {{ValuesConfig.cols}} x {{ValuesConfig.rows}}</h3>
+    <div class="board-table">
+      <table id="board"> 
+      </table>
+    </div>
+  </div>
+</template>
+
+<script>
+import {BoardBus} from '@/services/board-bus';
+export default {
+  name: 'board-bus',
+  data() {
+    return {
+      ValuesConfig: ''
+    }
+  },
+  mounted() {
+    BoardBus.$on('values-config', (value) => {
+      this.ValuesConfig = value;
+      this.drawBoard(value.cols, value.rows)
+    })
+  },
+  methods: {
+    drawBoard(cols, rows) {
+      let table = document.getElementById('board');
+      this.clearBoard(table);
+
+      for(var rowIndex = 0; rowIndex < rows; rowIndex++)
+      {
+       var x = table.insertRow(rowIndex);
+       for(var colIndex = 0; colIndex < cols; colIndex++)  
+        {
+          var y =  x.insertCell(colIndex);
+          //y.innerHTML="X";
+          y.classList.add("board-td");
+        }
+      }
+    },
+    clearBoard(table) {
+      while (table.hasChildNodes()) {
+        table.removeChild(table.lastChild);
+      }
```

Comment:
use vuejs directives instead of DOM manipulation, you could do something like:
```html
<table>
  <tr v-for="row in rows">
    <td v-for="column in columns"></td>
  </tr>
</table>
```

Reply:
This was changed in the last commit.